File: Cinemark/Persona.cpp
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

/* 
 * File:   Persona.cpp
 * Author: ASUS
 * 
 * Created on 4 de febrero de 2018, 19:25
 */

#include "Persona.h"

Persona::Persona(string nombre,int rut) {
    this->nombre = nombre;
    this->rut =  rut;
}

Persona::Persona(const Persona& orig) {
}

string Persona::getNombre() {
    return this->nombre;
}

void Persona::setNombre(string nombre) {
    this->nombre= nombre;
}

int Persona::getRut() {
    return this->rut;
}

void Persona::setRut(int rut) {
    this->rut=rut;
}




Persona::~Persona() {
}

File: ayudantia4-2019/src/ayudantia4/pkg2019/Persona.java
package ayudantia4.pkg2019;

/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

/**
 *
 * @author Juan Pablo Martinez
 */
public class Persona {
    private String nombre;
    private String rut;
    private  int cant;

    public Persona(String nombre, String rut, int cant) {
        this.nombre = nombre;
        this.rut = rut;
        this.cant = cant;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getRut() {
        return rut;
    }

    public void setRut(String rut) {
        this.rut = rut;
    }

    public int getCant() {
        return cant;
    }

    public void setCant(int cant) {
        this.cant = cant;
    }
    
}
